yagpdb-cc-website/docs/utilities/edit.md

```diff
@@ -1,8 +1,22 @@
 ---
-sidebar_position: 5
+sidebar_position: 6
 title: Edit
 ---
 
+This command is a tool for editing messages sent by YAGPDB.
+
+**Flags:**  -content : To Change Content  
+		-title, -desc, -image, -thumbnail, -url, -author, -authoricon, -authorurl, -footer, -footericon, -color : To Edit Embed  
+		-force : Makes a new embed with provided fields and discards previous embed (default behavior is simply editing provided fields of embed while preserving other fields)  
+		-clrembed : To remove the embed from a message previously containing embed (so that now it has only content. Note, You cant remove embed if content is also null)
```

Comment:
funky formatting here (I know it's probably from the original top comment) but we can fix it a bit here:
```suggestion
This command is a tool for editing messages sent by YAGPDB.

**Flags:**
`-content`: To Change Content  
`-title`, `-desc`, `-image`, `-thumbnail`, `-url`, `-author`, `-authoricon`, `-authorurl`, `-footer`, `-footericon`, `-color`: To Edit Embed  
`-force`: Makes a new embed with provided fields and discards previous embed (default behavior is simply editing provided fields of embed while preserving other fields)  
`-clrembed`: To remove the embed from a message previously containing embed (so that now it has only content. Note, You cant remove embed if content is also null)
```